blockstore.go

```diff
@@ -27,6 +27,7 @@ type exchangeBsWrapper struct {
 }
 
 func (e *exchangeBsWrapper) GetBlock(ctx context.Context, c cid.Cid) (blocks.Block, error) {
+	goLog.Debugw("block requested from saturn", "cid", c.String())
```

Comment:
@hacdias  is there a way to avoid executing [cid.String()](https://github.com/ipfs/go-cid/blob/06fe289a6a55f51664b104efdbb547120bc1696d/cid.go#L313-L330) when debug level is not active?

Reply:
We can, indeed. I'll add an `if`.